src/core/tests/Test_UbuntuProClient.py

```diff
@@ -13,10 +13,11 @@
 # limitations under the License.
 #
 # Requires Python 2.7+
-import imp
 import sys
 import types
 import unittest
+if sys.version_info < (3, 12):
```

Comment:
From this PR - https://github.com/Azure/LinuxPatchExtension/pull/306 
(Py 3.12 failure. That PR should ideally go in first.)